lib/static/new-ui/types/store.ts

```diff
@@ -1,51 +1,69 @@
 import {TestStatus, ViewMode} from '@/constants';
-import {BrowserItem, ImageFile} from '@/types';
+import {BrowserItem, ImageFile, ReporterConfig, TestError} from '@/types';
+import {HtmlReporterValues} from '@/plugin-api';
 
 export interface SuiteEntityNode {
+    id: string;
     name: string;
     status: TestStatus;
     suiteIds: string[];
+    suitePath: string[];
 }
 
 export interface SuiteEntityLeaf {
+    id: string;
     name: string;
     status: TestStatus;
     browserIds: string[];
+    suitePath: string[];
 }
 
 export type SuiteEntity = SuiteEntityNode | SuiteEntityLeaf;
 
 export const isSuiteEntityLeaf = (suite: SuiteEntity): suite is SuiteEntityLeaf => Boolean((suite as SuiteEntityLeaf).browserIds);
 
 export interface BrowserEntity {
+    id: string;
     name: string;
     resultIds: string[];
+    parentId: string;
 }
 
 export interface ResultEntityCommon {
+    id: string;
     parentId: string;
     attempt: number;
     imageIds: string[];
     status: TestStatus;
     timestamp: number;
+    metaInfo: Record<string, string>;
+    suiteUrl?: string;
 }
 
 export interface ResultEntityError extends ResultEntityCommon {
-    error: Error;
-    status: TestStatus.ERROR;
+    error: TestError;
+    status: TestStatus.ERROR | TestStatus.FAIL;
```

Comment:
Couldn't it also be "FAIL_ERROR" which you added in this PR?

Reply:
No it can't FAIL_ERROR is a status that's added by html-reporter, but we receive results from testplane